fiftyone/utils/qwen3_vl.py

```
@@ -217,8 +223,17 @@ class Qwen3VLModel(fout.TorchImageModel, fom.EmbeddingsMixin):
 
     def __init__(self, config):
         self._processor = None
+        self._mode = config.mode
         super().__init__(config)
 
+    @property
+    def mode(self):
+        return self._mode
+
+    @mode.setter
+    def mode(self, value):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `mode`

Add return type annotation: `None`

(ANN202)